presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/WeightedFairQueue.java

```diff
@@ -170,6 +171,15 @@ public int getUtilization()
         {
             return utilization;
         }
+
+        @Override
+        public String toString()
+        {
+            return toStringHelper(this)
+                    .add("share", share)
```

Comment:
now that you touched this class, we can mark `share` and `utilization` both `final`.